R/teal_lockfile.R

```diff
@@ -0,0 +1,90 @@
+#' Create `renv` `.lockfile`
```

Comment:
```suggestion
#' Create the application' `.lock` file
```

Reply:
However I would like to stick with `.lockfile` as this is the term I used all over the place.
So I will only change `renv` to `the application`

Reply:
And I will not use `the` as `the` and `a` are omitted in titles

Reply:
I was thinking that there is only a `.lock` file but actually there is a "lockfile" thing -> see: https://rstudio.github.io/renv/reference/lockfiles.html?q=lockfile#null

So maybe unquote this? Just to stay consistent.

Reply:
Ok now I see you changed your proposition to 

`Create the `.lock` file to reproduce the application environment`

I would go with

> Create `.lockfile` for project reproducibility

or 

> Create `.lockfile` for application reproducibility

So we keep it short